aiidalab_widgets_base/computational_resources.py

```diff
@@ -892,24 +900,18 @@ def on_setup_computer(self, _=None):
         )
         try:
             computer = computer_builder.new()
+            computer.store()
         except (
             ComputerBuilder.ComputerValidationError,
             common.exceptions.ValidationError,
         ) as err:
             self.message = f"{type(err).__name__}: {err}"
             return False
 
-        try:
-            computer.store()
-        except common.exceptions.ValidationError as err:
-            self.message = f"Unable to store the computer: {err}."
-            return False
-
-        if self._configure_computer(computer):
-            for function in self._on_setup_computer_success:
-                function()
-        self.message = f"Computer<{computer.pk}> {computer.label} created"
-        return True
+        if self._run_callbacks_if_computer_exists(self.label.value):
+            self.message = f"Computer<{computer.pk}> {computer.label} created"
+            return True
+        return False
```

Comment:
I think this is a correct suggestion, I add it. Although I think this part will never touch, if the computer setup fail it will caught by the try.except statement.